rskj-core/src/test/java/co/rsk/peg/federation/FederationSupportImplTest.java

```diff
@@ -2194,6 +2195,218 @@ void save_callsStorageProviderSave() {
         verify(storageProvider).save(federationMainnetConstants.getBtcParams(), activations);
     }
 
+    @Nested
+    @Tag("commit federation")
+    @TestInstance(TestInstance.Lifecycle.PER_CLASS)
+    class CommitFederationTests {
+        private ActivationConfig.ForBlock activations;
+
+        private FederationStorageProvider federationStorageProvider;
+        private FederationSupportImpl federationSupport;
+        private PendingFederation pendingFederation;
+        private BridgeEventLogger bridgeEventLogger;
+        
+        private Block rskExecutionBlock;
+        long rskExecutionBlockNumber = 1000L;
+        long rskExecutionBlockTimestamp = 10L;
+
+        @BeforeEach
+        void setUp() {
+            StorageAccessor bridgeStorageAccessor = new InMemoryStorage();
+            federationStorageProvider = new FederationStorageProviderImpl(bridgeStorageAccessor);
+            
+            pendingFederation = new PendingFederation(FederationTestUtils.getFederationMembers(5));
+            federationStorageProvider.setPendingFederation(pendingFederation);
+
+            bridgeEventLogger = mock(BridgeEventLogger.class);
+
+            rskExecutionBlock = mock(Block.class);
```

Comment:
Here as well. There is a BlockHeaderBuilder class that can help you create a Block object with the required properties

Reply:
sure, thanks

Reply:
done